src/functions/src/main/java/org/apache/jmeter/functions/LogFunction.java

```diff
@@ -175,8 +175,6 @@ static synchronized void logDetails(Logger logger, String stringToLog, String pr
                 case TRACE:
                     logger.trace("{} {} {}", threadName, separator, stringToLog, throwable);
                     break;
-                default:
```

Comment:
Do you really think we should protect from the case when `org.slf4j.event.Level` enum will have extra enum value in runtime?
I think it would be unlikely, and if we drop the default, then we'll know when slf4j adds new value in compile-time.